mifospay/src/main/java/org/mifos/mobilewallet/mifospay/home/ui/PaymentsFragment.java

```diff
@@ -40,7 +40,9 @@ public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
             Bundle savedInstanceState) {
         View rootView = inflater.inflate(R.layout.fragment_finance, container, false);
         ButterKnife.bind(this, rootView);
-
+        vpTabLayout.setPadding(5, 0,5, 0);
```

Comment:
@haran2248 these values as accepted as `pixels` so if you want to set `5dp` which I am guessing you probably are trying to, you need to first convert 5dp to pixels and then pass them here. 
Also always leave a space after `,`  I am not sure how this went pass through the GitHub actions we set up but anyways before making any PR please run `./gradlew check` or `gradlew check` on your machine to detect these errors on your machine only

Reply:
adding @s-ayush2903 here

Reply:
Oh i might have missed out on that,Ill make the corrections

Reply:
@devansh-299 While writing the actions I believed, that these mini-checks are included in our lint job, looks like it isn't so. No worries, it is an easy fix, I can push the changes needed in a new PR, thoughts?